src/include/meta.h

```diff
@@ -27,6 +27,7 @@
 #define	WT_METAFILE_URI		"file:WiredTiger.wt"	/* Metadata table URI */
 
 #define	WT_LAS_URI		"file:WiredTigerLAS.wt"	/* Lookaside table URI*/
+#define	WT_SYSTEM_URI		"system:info"		/* System info URI*/
```

Comment:
How about "system:checkpoint"?  If we want more "system" information in the metadata, I'd be inclined to keep it in separate records so we don't have to synchronize writes from different operation.